.github/workflows/deploy-infrastructure.yaml

```diff
@@ -49,7 +49,6 @@ jobs:
     permissions:
       id-token: write
       contents: read
```

Comment:
Why are we not using the timeout parameter ? And why are we not passing in an over-ride rather than changing the default  - is this not the one that all stacks will use ?